### ISSUE TITLE
Limpeza Banco de Dados

### ISSUE THREAD

**Issue by cdaher78** [ldurans/izing.io#221]
Estou terminando a fase de testes do Izing para colocar em produção e gostaria de levantar um banco de dados limpo.

Qual seria o procedimento?

**Comment by LuizMarroni** [comment]
Olá, acredito que seja isto para migrar e alimentar o banco de dados com os valores padrões:
npm run db:migrate
npm run db:seed

**Comment by ygorwil** [comment]
@LuizMarroni você tem algum e-mail pelo qual possamos conversar? Se você já está com o sistema rodando com todas as funcionalidades de forma correta, como parece o caso, e tiver alguns minutos pra tirar umas dúvidas te agradeço demais. Talvez possamos até fechar uma parceria caso eu consiga ao menos rodar o que é esperado à 100% já que provavelmente você entende um bocado sobre a plataforma e linguagens dela. Se preferir pode falar comigo pelo --------------.

Ótima quarta-feira e restante de semana a todos!

**Comment by LuizMarroni** [comment]
Claro! Sem problemas, lhe enviei um e-mail, se possível ver eu agradeceria!
